package.json

```diff
@@ -4,50 +4,63 @@
   "description": "Blazing Fast React UI Primitive",
   "contributors": [
     "Jeroen Ransijn (https://twitter.com/jeroen_ransijn)",
-    "Roland Warmerdam (https://roland.codes)"
+    "Roland Warmerdam (https://roland.codes)",
+    "Meichen Zhou (https://github.com/jfmaggie)",
+    "Netto Farah (https://twitter.com/nettofarah)"
   ],
   "keywords": [
     "react"
   ],
   "repository": "segmentio/ui-box",
   "license": "MIT",
-  "main": "lib/index.js",
+  "main": "lib/src/index.js",
   "files": [
-    "lib"
+    "lib/src"
   ],
   "sideEffects": false,
   "scripts": {
-    "test": "xo && nyc ava",
+    "test": "yarn run tslint && xo && nyc ava",
+    "tslint": "tslint -c tslint.json 'src/**/*.ts'",
     "prepublishOnly": "rm -rf lib && yarn run build",
     "precommit": "lint-staged",
-    "dev": "start-storybook -p 9009",
-    "build": "babel --out-dir lib src --source-maps inline",
+    "dev": "start-storybook -p 9009 -c .storybook",
+    "build": "tsc --declaration",
     "build-storybook": "build-storybook -s .storybook/static -o .out",
     "release": "np --no-publish",
     "benchmark": "echo ui-box && react-benchmark tools/benchmarks/box.js && echo glamor/jsxstyle && react-benchmark tools/benchmarks/glamor-jsxstyle.js",
     "size": "size-limit",
     "coverage": "nyc report --reporter=html"
   },
   "dependencies": {
-    "@emotion/hash": "^0.6.5",
     "glamor": "^2.20.0",
     "inline-style-prefixer": "^5.0.1",
+    "murmurhash-js": "^1.0.0",
```

Comment:
I used `@emotion/hash` because `murmurhash-js` isn't maintained (last updated 5 years ago).

Reply:
I also think the [Emotion](https://emotion.sh/) team made some performance improvements to the original implementation.

Reply:
I mean... it is a murmur hash, not very complex / shouldn't need changes